### DISCUSSION TITLE
Create user with access to all topic and can produce and consume but can not create new user with acl

### DISCUSSION THREAD

**Discussion by arezvani** [strimzi/strimzi-kafka-operator#8127] [Q&A] [answered]
Hi,
I want to create user to access all topics in cluster and not be super-user.
I use `patternType: prefix` and in name field, I try these values:

* `name: "*"`
* `name: ""`

but when I list my topics, the result will be empty.

What should I do? Please help me

**Comment by scholzj** [comment]
You should probably start a full custom resource for a start to make it more clear what you tried.

**Comment by sreejesh-radhakrishnan-db** [comment, marked as answer]
Hi I am also seeing same issue-  I have upgraded from 0.28 to 0,32. and  I am creating a new user  -with below manifest

apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaUser
metadata:
  name:  907100
  labels:
    strimzi.io/cluster: test
spec:
  authentication:
    type: tls-external
  authorization:
    type: simple
    acls:            
      - resource:
          type: topic
          name:  "*"
          patternType: literal
        operations:
          - Read
          - Write
          - Create
          - Delete
          - Alter
          - Describe
          - ClusterAction
          - AlterConfigs
          - DescribeConfigs
          - IdempotentWrite
          - All
        host: "*"

      - resource:
          type: group
          name: "*"
          patternType: literal
        operations:
          - Read
          - Write
          - Create
          - Delete
          - Alter
          - Describe
          - ClusterAction
          - AlterConfigs
          - DescribeConfigs
          - IdempotentWrite
          - All
        host: "*"
      - resource:
          type: transactionalId
          name: "*"
          patternType: literal
        operations:
          - Read
          - Write
          - Create
          - Delete
          - Alter
          - Describe
          - ClusterAction
          - AlterConfigs
          - DescribeConfigs
          - IdempotentWrite
          - All
        host: "*"

**Comment by sreejesh-radhakrishnan-db** [comment]
I am really confused,  with above manifest I am getting issue.  the user is shown as READY status , but still not able to connect to topic.  Hence I shared the manifest. Some how its marked as answer. not sure  how..  @scholzj  please can you check manifest and confirm if look good?

**Reply by scholzj** [reply]
The manifest is badly formatted, so I do not know if it looks good or not. I'm not really sure why @arezvani selected it as the correct answer. Maybe it helped him in some way with his problem I guess?
